anuvaad-etl/anuvaad-extractor/file_translator/etl-file-translator/services/service.py

```diff
@@ -113,18 +115,33 @@ def get_runs(self, iterable_obj, para_obj=False, run_obj=False, run_lst=False, f
         if para_obj and file_type == config.TYPE_PPTX:
             return iterable_obj.runs
         if para_obj:
-            runs = []
+            runs = [[] for i in range(20)]
```

Comment:
we have discussed this approach right that we will not create a list of list before hand